test/functional/unified-spec-runner/operations.ts

```diff
@@ -270,6 +270,12 @@ operations.set('findOneAndUpdate', async ({ entities, operation }) => {
   return (await collection.findOneAndUpdate(filter, update, translateOptions(opts))).value;
 });
 
+operations.set('findOneAndDelete', async ({ entities, operation }) => {
+  const collection = entities.getEntity('collection', operation.object);
+  const { filter, ...opts } = operation.arguments;
+  return (await collection.findOneAndDelete(filter, opts)).value;
```

Comment:
Just curious, how did this work before without extracting the value?

Reply:
I think we just didn't have any unified tests for this operation yet.